data/indicator.css

```
@@ -30,6 +40,14 @@
     -gtk-icon-source: -gtk-icontheme("microphone-sensitivity-muted-symbolic");
 }
 
+@keyframes volume-blocking {
+    0% { -gtk-icon-transform: rotate(20deg); }
+    25% { -gtk-icon-transform: rotate(-20deg); }
+    50% { -gtk-icon-transform: rotate(10deg); }
+    75% { -gtk-icon-transform: rotate(-10deg); }
+    100% { -gtk-icon-transform: rotate(5deg); }
+}
```

I like it in general, but the rotation feels weird to me since that icon is not visually weighted in the center. What about a side-to-side wiggle?

```suggestion
@keyframes volume-blocking {
    20% { -gtk-icon-transform: translateX(4px); }
    40% { -gtk-icon-transform: translateX(-4px); }
    60% { -gtk-icon-transform: translateX(2px); }
    80% { -gtk-icon-transform: translateX(-2px); }
    100% { -gtk-icon-transform: translateX(1px); }
}
```